### PR TITLE
Add user response submission

### PR DESCRIPTION
**Pull request by JackieSL1**
# Description

Add survey responses for a single comment question
* Responses for multiple questions will be added next in a separate PR

## Issues

#58 

# How to test

Manual testing:
1. Create a new survey with a comment response question
2. Open the survey
3. Go to the survey response page and hit submit

Integration tests:
* Run `mvn test`

# Checklist
**I've added tests**
- [x] Yes

- [ ] No

**I've updated the DB schema and UML diagram**
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/java/sysc4806/project24/mini_survey_monkey/controllers/RespondController.java [hunk 1]
```diff
@@ -21,7 +24,26 @@ public RespondController(SurveyRepository surveyRepository) {
     public String respond(@PathVariable("surveyID") int surveyID, Model model) {
         Survey survey = surveyRepository.findById(surveyID);
         model.addAttribute("survey", survey);
+        // TODO: Handle form having no questions
```
Comment (AliAbdollahian) [inline src/main/java/sysc4806/project24/mini_survey_monkey/controllers/RespondController.java:27]: maybe adding if statement to check if there is no question then we redirect to a page where tell user no question has been added

Comment (JackieSL1) [reply src/main/java/sysc4806/project24/mini_survey_monkey/controllers/RespondController.java:27]: Do you think it's okay if I add this in a future PR?

File: src/test/java/sysc4806/project24/mini_survey_monkey/integration/ResponseTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import static org.hamcrest.Matchers.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.model;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class ResponseTest {
```
Comment (AliAbdollahian) [inline src/test/java/sysc4806/project24/mini_survey_monkey/integration/ResponseTest.java:22]: run successfully and it looks good to me

File: src/main/java/sysc4806/project24/mini_survey_monkey/controllers/RespondController.java (hunk 1 above)
Comment (AliAbdollahian) [reply src/main/java/sysc4806/project24/mini_survey_monkey/controllers/RespondController.java:27]: Yes definitely

**Review by AliAbdollahian — APPROVED** [review]
Nice job thanks @JackieSL1

**Comment by JackieSL1** [comment]
> Nice job thanks @JackieSL1

Thanks, Ali!

**Pull request merged**